sdk/typescript/src/types/committee.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Heads up: there's a PR from the community for the same endpoint: https://github.com/MystenLabs/sui/pull/6388/files. Feel free to remove from your PR